content/3_Silver/2P.mdx

```diff
@@ -31,6 +31,133 @@ frequency: 2
 	</Resource>
 </Resources>
 
+<FocusProblem problem="book" />
+
+### Solution - Books
+
+We want to find the longest continuous segment of books that can be read within $t$ minutes. 
+
+To accomplish this, we can define $\texttt{left}$ and $\texttt{right}$ as the start and end of the segment respectively.
+
```

Comment:
Think it would help to add another line saying Since being able to read segment $\texttt{left} \dots \texttt{right}$ implies we can read segments $\texttt{left} \dots \texttt{right}-1$, we can use two-pointer (or, alternatively, binary search)